imports/ui/QuestionEditItem.jsx

```diff
@@ -224,27 +221,31 @@ export class QuestionEditItem extends Component {
    * Set CourseId and add corresponding tag (called from dropdown)
    * @param {course} e
    */
-   setCourse (e) {
-     let cId = e.target.value
-     if (parseInt(cId) !== -1) {
-       let tags = this.state.tags
-       Meteor.call('courses.getCourseCodeTag', cId, (error, tag) => {
-          tlabels = _(tags).pluck('label')
-          if (tag && !tlabels.includes(tag.label)){
+  setCourse (e) {
```

Comment:
This should be tested,  as it can be a little fragile as it also changes the tags

Reply:
The issue here is that there was no handling of the error during the method call. Elsewhere in the file (and across the project), this is how it is handled. If this method should be called no matter what (even during an error) it's something we should discuss.

Reply:
In upcoming iteration, I replaced the if-else block with an if return.

From a logic perspective, does it make sense save the question even if there's an error retrieving the tags?